docs/userguide/pyproject_config.rst

```diff
@@ -70,6 +70,14 @@ The ``project`` table contains metadata fields as described by the
    # ... other project metadata fields as listed in:
    #     https://packaging.python.org/en/latest/guides/writing-pyproject-toml/
 
+.. important::
+   Support for
+   :external+PyPUG:ref:`project.license-files <pyproject-toml-license-files>`
```

Comment:
Just wondering, would this be a better link target here?
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-files

```suggestion
   Support for
   :external+PyPUG:ref:`project.license-files <license-file>`
```

Reply:
Oh, no, I did not see this page. I ended up openning a PR 
 to the other one, https://github.com/pypa/packaging.python.org/pull/1832 so that we could have more specific targets.